### ISSUE TITLE
Test tools.tests.test_numdiff.TestDerivativeFun.test_hess_fun1_fd failing

### ISSUE THREAD

**Issue by sergiopasra** [statsmodels/statsmodels#1319]
I'm submitting here a new error I've found while creating the Fedora package (still pending [review](https://bugzilla.redhat.com/show_bug.cgi?id=1053640)). #1316 is related with this bug

The test statsmodels.tools.tests.test_numdiff.TestDerivativeFun.test_hess_fun1_fd is
failing in Fedora 20 with the following versions

```
In [2]: sm.version.full_version
Out[2]: '0.6.0.dev-b401d4d'In 

In [6]: sm.api.show_versions()

INSTALLED VERSIONS
------------------
Python: 2.7.5.final.0
OS: Linux 3.12.7-300.fc20.x86_64 #1 SMP Fri Jan 10 15:35:31 UTC 2014 x86_64
byteorder: little
LC_ALL: None
LANG: es_ES.utf8

Statsmodels
===========

Installed: 0.6.0.dev-b401d4d (statsmodels)

Required Dependencies
=====================

cython: 0.19.2 (/usr/lib64/python2.7/site-packages/Cython)
numpy: 1.8.0 (/usr/lib64/python2.7/site-packages/numpy)
scipy: 0.12.1 (/usr/lib64/python2.7/site-packages/scipy)
pandas: 0.10.0 (/usr/lib64/python2.7/site-packages/pandas)
    dateutil: 1.5 (/usr/lib/python2.7/site-packages/dateutil)
patsy: 0.2.1 (/usr/lib/python2.7/site-packages/patsy)

Optional Dependencies
=====================

matplotlib: 1.3.0 (/usr/lib64/python2.7/site-packages/matplotlib)
cvxopt: Not installed

Developer Tools
================

IPython: 0.13.2 (/usr/lib/python2.7/site-packages/IPython)
    jinja2: 2.7.1 (/usr/lib/python2.7/site-packages/jinja2)
sphinx: 1.1.3 (/usr/lib/python2.7/site-packages/sphinx)
    pygments: 1.6 (/usr/lib/python2.7/site-packages/pygments)
nose: 1.3.0 (/usr/lib/python2.7/site-packages/nose)
virtualenv: 1.10.1 (/usr/lib/python2.7/site-packages)

```

And the stack trace is

```
======================================================================
FAIL: statsmodels.tools.tests.test_numdiff.TestDerivativeFun.test_hess_fun1_fd
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/spr/devel/statsmodels/build/lib.linux-x86_64-2.7/statsmodels/tools/tests/test_numdiff.py", line 217, in test_hess_fun1_fd
    assert_almost_equal(hetrue, hefd, decimal=DEC3)
  File "/usr/lib64/python2.7/site-packages/numpy/testing/utils.py", line 454, in assert_almost_equal
    return assert_array_almost_equal(actual, desired, decimal, err_msg)
  File "/usr/lib64/python2.7/site-packages/numpy/testing/utils.py", line 811, in assert_array_almost_equal
    header=('Arrays are not almost equal to %d decimals' % decimal))
  File "/usr/lib64/python2.7/site-packages/numpy/testing/utils.py", line 644, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 3 decimals

(mismatch 11.1111111111%)
 x: array([[ 0.,  0.,  0.],
       [ 0.,  0.,  0.],
       [ 0.,  0.,  0.]])
 y: array([[-0.00076004, -0.0010625 , -0.00143276],
       [-0.0010625 , -0.00078563, -0.00106824],
       [-0.00143276, -0.00106824, -0.00153653]])

----------------------------------------------------------------------
```

**Comment by josef-pkt** [comment]
What was going on with this?     issue got lost because it didn't have labels.

I haven't seen a failure like this in a long time, but I'd like to know if there was a real problem or bug and what fixed it.

**Comment by sergiopasra** [comment]
It seems to be fixed in 0.6, at leat, I don't get any errors running the tests. I don't know how it was fixed. It would be interesting to know to patch exiting 0.5 rpm packages in older releases of Fedora

**Comment by josef-pkt** [comment]
this is old, and it seems to work.

(I have no idea what change from 0.5 to 0.6 related to this)
